From 6b68ba1be1931affb558713af303f27f400e7229 Mon Sep 17 00:00:00 2001
From: Paolo Velati <paolo.velati@gmail.com>
Date: Mon, 17 Apr 2023 20:27:28 +0000
Subject: [PATCH] hwdb: Fix rotation for BMAX Y13

(cherry picked from commit d5fbaa965e925f146363d2ae8ccad90bcbb5960d)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 038e9dc013..ebf8c718a7 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -192,6 +192,15 @@ sensor:modalias:acpi:BMI0160*:dmi:*:svnAYANEO:pnAIR*:*
 sensor:modalias:acpi:BMI0160*:dmi:*:svnAYANEO:pn*NEXT*:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
+#########################################
+# BMAX
+#########################################
+
+# BMAX Y13
+sensor:modalias:acpi:KIOX010A:*:dmi:*:svnAMI:*:skuH2M6:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, -1, 0; 0, 0, -1
+ ACCEL_LOCATION=display
+
 #########################################
 # Chuwi
 #########################################
